src/measure_theory/measure/null_measurable.lean

```diff
@@ -109,7 +109,7 @@ protected lemma congr (hs : null_measurable_set s μ) (h : s =ᵐ[μ] t) :
   null_measurable_set t μ :=
 let ⟨s', hm, hs'⟩ := hs in ⟨s', hm, h.symm.trans hs'⟩
 
-protected lemma Union [encodable ι] {s : ι → set α}
+protected lemma Union [countable ι] {s : ι → set α}
```

Comment:
It would be nice to have this and similar lemmas work for `Sort *` but this could wait for another PR.

Reply:
Some generalized without change to the proof, I fixed one proof to generalize, and I can try some of the remaining ones.